### PR TITLE
Added Moja global's Branding Guidelines

### PR DESCRIPTION
**Pull request by vedant-kakde**
Signed-off-by: Vedant Kakde <69970950+vedant-kakde@users.noreply.github.com>

## Description

Added Moja global's new branding guidelines on community-website.

## Type of change

- [x] This change requires a documentation update.

...

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] I have added corresponding tests.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: website/docs/02-About/3-branding.mdx [hunk 1]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
 
 **Moja global** is always written in all lower case, except when at the start of a sentence in which case it appears as **Moja global**. Moja global is written in lower case because **open source** is also the lower case (even though both are registered trademarks). While there is no published reason why moja global uses lower case, some have explained the use of lower case as a contrast to proper nouns and as an expression of the supportive role moja global tries to fulfill.
 
-### Logo
+## Logomarks
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:15]: Give a line break between headings.

File: website/docs/02-About/3-branding.mdx [hunk 2]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/digital-correspondence.png"/>
+
+### Presentations and Data visualizations
+
+<img src="/img/brand/presentations-and-data-visualizations.png"/>
+
+### Social Media and Training Platforms
+
+<img src="/img/brand/social-media-and-training-platforms.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:173]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 3]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/printed-stationery.png"/>
+
+### Digital Correspondence
+
+<img src="/img/brand/digital-correspondence.png"/>
+
+### Presentations and Data visualizations
+
+<img src="/img/brand/presentations-and-data-visualizations.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:169]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 4]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/outdoor-marketing.png"/>
+
+### Printed Stationery
+
+<img src="/img/brand/printed-stationery.png"/>
+
+### Digital Correspondence
+
+<img src="/img/brand/digital-correspondence.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:165]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 5]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/website-1.png"/>
+
+### Outdoor Marketing
+
+<img src="/img/brand/outdoor-marketing.png"/>
+
+### Printed Stationery
+
+<img src="/img/brand/printed-stationery.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:161]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 6]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+### Website
+
+<img src="/img/brand/website.png"/>
+
+<img src="/img/brand/website-1.png"/>
+
+### Outdoor Marketing
+
+<img src="/img/brand/outdoor-marketing.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:157]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 7]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/imagery.png"/>
+
+## Example Applications
+
+### Website
+
+<img src="/img/brand/website.png"/>
+
+<img src="/img/brand/website-1.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:153]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 8]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+## Imagery
+Imagery follows the similar richness and nature focus as the brand color palette with subject matter comprised of aerial landscape and macro photography in key regions, diversity and human interaction with nature.
+
+<img src="/img/brand/imagery.png"/>
+
+## Example Applications
+
+### Website
+
+<img src="/img/brand/website.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:151]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 9]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+### Color Gradients
+
+Color gradients represent the power of collaboration and can be used in application as background colours and as seen in the brand graphics.
+
+<img src="/img/brand/color-gradients.png"/>
+
+## Imagery
+Imagery follows the similar richness and nature focus as the brand color palette with subject matter comprised of aerial landscape and macro photography in key regions, diversity and human interaction with nature.
+
+<img src="/img/brand/imagery.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:145]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 10]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+The brand colors communicate earth and the natural environment whilst celebrating the innovation of the FLINT software and collaborative spirit of the community.
+
+<img src="/img/brand/color-palette.png"/>
+
+### Color Gradients
+
+Color gradients represent the power of collaboration and can be used in application as background colours and as seen in the brand graphics.
+
+<img src="/img/brand/color-gradients.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:140]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 11]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+This page exemplifies the sizing and proportions of various styles of text.
+
+<img src="/img/brand/typographic-hierarchy.png"/>
+
+## Color
+### Color Palette
+
+The brand colors communicate earth and the natural environment whilst celebrating the innovation of the FLINT software and collaborative spirit of the community.
+
+<img src="/img/brand/color-palette.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:134]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 12]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+### Typographic Hierarchy
+
+Hierarchy of text is important in a design because it allows the reader to understand, at a glance, the order of importance of the information at hand.
+
+This page exemplifies the sizing and proportions of various styles of text.
+
+<img src="/img/brand/typographic-hierarchy.png"/>
+
+## Color
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:129]: Give a line break after headings

File: website/docs/02-About/3-branding.mdx [hunk 13]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/typographic-families.png"/>
+
+### Typographic Hierarchy
+
+Hierarchy of text is important in a design because it allows the reader to understand, at a glance, the order of importance of the information at hand.
+
+This page exemplifies the sizing and proportions of various styles of text.
+
+<img src="/img/brand/typographic-hierarchy.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:127]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 14]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+### Typographic Families
+
+DM Sans is a low-contrast geometric sans serif design, chosen for it's legibility and approachable rounded forms.
+
+It is a free font commissioned by Google from Colophon, an international and award-winning type foundry based in London (UK) and Los Angeles (US)
+
+[Download DM Sans](https://fonts.google.com/specimen/DM+Sans?preview.text_type=custom)
+
+<img src="/img/brand/typographic-families.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:119]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 15]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+In application, an 8-column grid should be used and graphics should be applied at a 2-column, 3-column, or 4-column scale.
+
+Graphics can be placed liberally to accommodate other content and can be used fully or partially on the page.
+
+<img src="/img/brand/placement-and-usage.png"/>
+
+
+Example application: outdoor marketing
+
+<img src="/img/brand/outdoor-marketing.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:107]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 16]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/graphic-gradients.png"/>
+
+### Placement and usage
+
+In application, an 8-column grid should be used and graphics should be applied at a 2-column, 3-column, or 4-column scale.
+
+Graphics can be placed liberally to accommodate other content and can be used fully or partially on the page.
+
+<img src="/img/brand/placement-and-usage.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:102]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 17]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/graphic-elements.png"/>
+
+### Graphic Gradients
+
+In application, graphics further suggest these values through the merging and collaborating of color through gradients.
+
+These graphics can be used on their own as symbolism or together to interact with other content.
+
+<img src="/img/brand/graphic-gradients.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:94]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 18]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+8. Do not angle the logo 09. Do not use the logo in an unspecified colour
+
+<img src="/img/brand/logomark-misuse.png"/>
+
+## Graphic Elements
+A suite of brand graphics are the ore of the Moja Global visual system and collectively work to represent the pillars of the brand: collaboration, innovation, flexibility and accessibility.
+
+Shapes combine and overlap in various configurations, whilst a bright pop of 'collaboration green' is used to highlight where shapes overlay, signifying the power of collaboration.
+
+<img src="/img/brand/graphic-elements.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:86]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

File: website/docs/02-About/3-branding.mdx [hunk 19]
```diff
@@ -12,33 +12,162 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+1. Do not remove any asset of the logo
+2. Do not distort or skew the logo
+3. Do not modify the composition or its proportions
+4. Do not outline the typography or icon
+5. Do not add drop shadow/filters
+6. Do not frame the logo
+7. Do not use the logo on a background colour that renders any part of logo illegible.
+8. Do not angle the logo 09. Do not use the logo in an unspecified colour
+
+<img src="/img/brand/logomark-misuse.png"/>
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:79]: Provide proper alt text and use an image caption. Refer `Moja global's mission` to know how it is done.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Round-1 Review

File: website/docs/02-About/3-branding.mdx [hunk 20]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+#### Monochromatic Usage
+
+Shown here are the monochromatic versions of the primary logomark. They are to be only used in cases where color print is restricted.
+
+| Description  | Image |
+| ------------- |:-------------:|
+| Secondary Colorway 2A - is to be used in instances where a white/light background is present.      | <img src="/img/brand/secondary-logomark-mono-2a.png" alt="secondary logomark mono 2a" width="150"/> |
+| Secondary Colorway 2B - is to be used in instances where a black/dark background is present.      | <img src="/img/brand/secondary-logomark-mono-2b.png" alt="secondary logomark mono 2b" width="150"/> |
+
+## Logomark Usage
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:57]: Give a line break between headings

File: website/docs/02-About/3-branding.mdx [hunk 21]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
 **Moja global** is always written in all lower case, except when at the start of a sentence in which case it appears as **Moja global**. Moja global is written in lower case because **open source** is also the lower case (even though both are registered trademarks). While there is no published reason why moja global uses lower case, some have explained the use of lower case as a contrast to proper nouns and as an expression of the supportive role moja global tries to fulfill.
 
-### Logo
+## Logomarks
 
-| Name | Image |
-| --- | --- |
-| Logo (Transparent Background) | <img src="/img/moja-global-logo-transparent-background.png" width="100"/> |
-| Logo (White Background) | <img src="/img/moja-global-logo-white-background.png" width="100"/> |
+### Primary Logomark
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:17]: `Logomark` should be in smallcase. Headings should always have the first character in the uppercase alone.

File: website/docs/02-About/3-branding.mdx [hunk 22]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
 
-- **Primary Typeface**: Source Sans pro (Used on the [Website](https://moja.global/))
-- **Secondary Typeface**: Ropa Sans (Used on the Presentation)
+| Description  | Image |
+| ------------- |:-------------:|
+| Primary Colorway 1A is to be used in instances where a white/light colored background and/or a light colored image is present.      | <img src="/img/brand/primary-logomark-1a.png" alt="primary logomark 1a" width="150"/>     |
+| Primary Colorway 1B is to be used in instances where a black/dark colored background and/or a light colored image is present.      | <img src="/img/brand/primary-logomark-1b.png" alt="primary logomark 1b" width="150"/>     |
 
-Secondary Typeface is used only when the Primary Typeface is not available.
+#### Monochromatic Usage
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:28]: Replace `Usage` with `usage`

File: website/docs/02-About/3-branding.mdx [hunk 23]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
 
-- **Main Color**: `#699b98`
-- **Accent Color 1**: `#808080`
+| Description  | Image |
+| ------------- |:-------------:|
+| Primary Colorway 1A - is to be used in instances where a white/light background is present.      | <img src="/img/brand/primary-logomark-mono-1a.png" alt="primary logomark mono 1a" width="150"/> |
+| Primary Colorway 1B is to be used in instances where a black/dark background is present.      | <img src="/img/brand/primary-logomark-mono-1b.png" alt="primary logomark mono 1b" width="150"/> |
 
-The Accent Color is to be rarely used. The Main Color should be used across Font, Banners, and Menus.
+### Secondary Logomark
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:37]: Replace `Logomark` with `logomark`

File: website/docs/02-About/3-branding.mdx [hunk 24]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
 
-### Imagery
+Shown here is the secondary Moja Global logomark.
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:39]: Replace `Global` with `global` and `Moja` with `moja`

File: website/docs/02-About/3-branding.mdx [hunk 25]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
 
-The images should be high-quality, engaging photography that portrays images of landscapes, forests, plants, and communities, and individuals interacting with the landscape.
+The secondary logomark should be used carefully and in cases where the brand graphics are already present on the page.
 
-An example shown below is included in the [images](https://github.com/moja-global/About_moja_global/blob/master/images) directory. Please ensure that proprietary photographs are credited if the source is known.
+| Description  | Image |
+| ------------- |:-------------:|
+| Secondary Colorway 2A - is to be used in instances where a white/light colored background and/or a light colored image is present.      | <img src="/img/brand/secondary-logomark-2a.png" alt="secondary logomark 2a" width="150"/> |
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:45]: The alt-text should be descriptive of the image itself. Some of the alt-text you wrote is quite good and could even be incorporated into the text, but it needs to be reformulated into actual alt-text for the images. [Here's a nice guide about writing alt-text](https://webaim.org/techniques/alttext/).

File: website/docs/02-About/3-branding.mdx [hunk 26]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+## Logomark Usage
+### Clear Space and Sizing
+Both the primary and secondary logos should always appear in their entirety, surrounded by a minimum area of clear space, so that headlines, text or other visual elements do not encroach on the logo. The minimum clear space is shown using the height of the brand icon. Logos should be used at a minimum size of 60px or 30mm (w).
+
+<img src="/img/brand/clear-space-and-sizing.png" alt="Clear Space and Sizing" />
+
+### Logomark Misuse
+
+Do's and Don'ts
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:65]: Remove this line

File: website/docs/02-About/3-branding.mdx [hunk 27]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+3. Do not modify the composition or its proportions
+4. Do not outline the typography or icon
+5. Do not add drop shadow/filters
+6. Do not frame the logo
+7. Do not use the logo on a background colour that renders any part of logo illegible.
+8. Do not angle the logo 09. Do not use the logo in an unspecified colour
+
+<img src="/img/brand/logomark-misuse.png"/>
+
+## Graphic Elements
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:82]: Give a line break

File: website/docs/02-About/3-branding.mdx [hunk 28]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+<img src="/img/brand/clear-space-and-sizing.png" alt="Clear Space and Sizing" />
+
+### Logomark Misuse
+
+Do's and Don'ts
+
+Take care to maintain the integrity of the logomark. The rules demonstrated here apply to all versions of logos.
+
+Do not:
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:69]: Use full sentences

File: website/docs/02-About/3-branding.mdx [hunk 29]
```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
...
+
+### Placement and usage
+
+In application, an 8-column grid should be used and graphics should be applied at a 2-column, 3-column, or 4-column scale.
+
+Graphics can be placed liberally to accommodate other content and can be used fully or partially on the page.
+
+<img src="/img/brand/placement-and-usage.png" alt="Placement and usage" />
+
+
```
Comment (HarshCasper) [inline website/docs/02-About/3-branding.mdx:105]: Remove the extra line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by sohamsshah** [comment]
Hey @vedant-kakde , are you working on this?

**Comment by vedant-kakde** [comment]
@sohamsshah Yes. Please allow me some time.

**Comment by sohamsshah** [comment]
> @sohamsshah Yes. Please allow me some time.

Sure! Please take your time. Just was asking as a follow-up :)

**Review by helper-uttam — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by vedant-kakde** [comment]
Thanks @HarshCasper !

**Pull request merged**